packages/plugins/Example/package.json

```diff
@@ -9,8 +9,8 @@
     }
   },
   "dependencies": {
-    "@masknet/plugin-infra": "workspace:^",
-    "@masknet/shared-base": "workspace:^",
+    "@masknet/plugin-infra": "workspace:*",
+    "@masknet/shared-base": "workspace:*",
```

Comment:
Why? Does this change necessary?

Reply:
No need indeed. Just an attempt to fix CI.